harness/determined/common/experimental/checkpoint/_checkpoint.py

```diff
@@ -132,13 +126,17 @@ def download(self, path: Optional[str] = None) -> str:
         if not any(p.exists() for p in potential_metadata_paths):
             # If the target directory doesn't already appear to contain a
             # checkpoint, attempt to fetch one.
-            if self.experiment_config["checkpoint_storage"]["type"] == "shared_fs":
-                src_ckpt_dir = self._find_shared_fs_path()
+            if not self.training:
```

Comment:
lgtm, but as a nitpick:
```suggestion
            if self.training is None:
```